### ISSUE TITLE
LDRS 399 video access

### ISSUE THREAD

**Issue by kmarjanovic** [TWUOnline/bug-tracking#1005]
### Course
[LDRS 399: Special Topics in Leadership (FAR MASTER)](https://learn.twu.ca/course/view.php?id=15742&section=0#tabs-tree-start)

### Instructions

Shine is revising LDRS 399 and is requesting access to the videos.  They should be fully available for anyone logged into the course, as Shine is, so I’m not sure why she can’t access them.

It looks like those videos are from Melinda’s SharePoint MS Streams account.  Do we have access - or a copy of these videos?

e.g.  [Audience and Purpose-20200710_095113.mov](https://mytwu-my.sharepoint.com/personal/melindad_twu_ca/_layouts/15/stream.aspx?id=%2Fpersonal%2Fmelindad%5Ftwu%5Fca%2FDocuments%2FStream%20Migrated%20Videos%2FAudience%20and%20Purpose%2D20200710%5F095113%2Emov&referrer=StreamWebApp%2EWeb&referrerScenario=AddressBarCopied%2Eview%2E5067404b%2De172%2D46e9%2D9e50%2D937105d038d2)

https://github.com/TWUOnline/ba-lead/blob/master/pages/14.ldrs399/07.U1/_1-3/alert.md

**Comment by chloecwh** [comment]
I'll let @Jeff-TWU answers this question ☺️

**Comment by Jeff-TWU** [comment]
Hi @kmarjanovic For 2 of the videos I was able to get the new MS Stream embed codes.

The course page embed links were from the old MS Stream and needed to be replaced with embed links from the new MS Stream.

Unfortunately I cannot see video for activity 1.4. It's possible the video was deleted or moved. Since I don't have edit access, I can only guess.

Also, although I updated the course material on Github, and flushed the cache on Grav, and cleared my browser cache, I'm still not seeing the embed changes I made. Did not see the changes in Grav, but confirmed they were made in Github (So I think Grav is being silly (that's a different issue @cmadland). 

In terms of specific access to new MS Stream files (editing or moving), I don't have that. Unlike the old version of MS Stream (which had super user privileges) only the owner, and presumably IT has control of files on Sharepoint/Onedrive.

If Shine is wanting edit access for another users videos, she will need to request access from the owner, Melinda, or if that is not feasible, then the request needs to be passed to IT.

I am curious to know if the automatically created redirects from the old version of MS Stream are able to keep track of a video file if a user moves the file in Onedrive @barishgolland.
